fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1218,4 +1218,24 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean recover_with_empty_tablet = false;
-}
+
+    /**
+     * Sql_result_cache
+     * Whether or not the result cache is enabled in Fe level, it can be overwritten with connection/session
+     * level setting in Context.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_result_cache = false;
+
+    /**
+     * Specify how long an entry will be expired in milliseconds, 10000 by default.
+     */
+    @ConfField(mutable = true)
+    public static long result_cache_expire_after_in_milliseconds = 10*1000;
+
+    /**
+     * Specify the overall threshold of local cache in bytes, 1G bytes by default.
+     */
+    @ConfField(mutable = true)
+    public static long result_cache_size_in_bytes = 1024 * 1024 * 1024;
```

Comment:
you should also limit ervey cache result entity size to avoid one large result use all caches,  we prefer to cache small size and need more time to calculating result,